check-deployment.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
Did we have any issue with aws s3 sync command ? I do not think this check is necessary . 
If the upload fails for any reason it propagates the correct exit Code

Reply:
I introduced due to 2-step process but with sync, it's now removed.